odhseldon/README.md

```diff
@@ -0,0 +1,20 @@
+# Seldon
+
+Seldon component subscribes to Seldon certified operator, which in turn issues an install of the Seldon certified operator in the namespace. 
+ 
+### Folders
+There are is one main folders in the Seldon component
+1. cluster: contains the subscription to the Seldon operator
+
+
+
+### Installation
+To install Prometheus add the following to the `kfctl` yaml file.
```

Comment:
We're not installing Prometheus here.....copy paste, eh?? :)

Reply:
lol fixed